dashboard/common/db_util.py

```diff
@@ -136,4 +137,4 @@ def get_canvas_data_date():
                 return date
     except Exception:
         logger.info("Value could not be found from metadata", exc_info = True)
-    return datetime.min
\ No newline at end of file
+    return datetime.min
```

Comment:
I don't see any code changes associated with this file can the changes be undone?